libpod/pod_internal.go

```diff
@@ -179,3 +180,10 @@ func startNode(ctx context.Context, node *containerNode, setError bool, ctrError
 
 	return
 }
+
+func (p *Pod) pauseContainerID() (string, error) {
```

Comment:
We probably don't need this once you move to just storing the pause container ID as the dependency for containers, and don't need to look it up anymore.

Reply:
What are you saying should change?

Reply:
pod inspect uses the internal version, but I could just access with p.config.pauseContainerID . I feel like I'd like to keep it but am open to changing

Reply:
Now that the updatePod is gone, it's better to just hit p.state.pauseContainerID